lib/testing/src/lib/process-services/pages/start-process.page.ts

```diff
@@ -145,9 +152,21 @@ export class StartProcessPage {
         return new FormFields();
     }
 
+    async checkSelectProcessField(): Promise<void> {
+        await BrowserVisibility.waitUntilElementIsNotVisible(this.disabledSelectProcessDropdown);
+    }
+
     async startProcess({name, processName }) {
```

Comment:
avoid use object in the firm of the method use two input parameters, we never use this kind of declaration

Reply:
This is how it was implemented previously. I've seen this method it's not much used, so i can refactor.